Lib/test/test_typing.py

```diff
@@ -4967,6 +5200,16 @@ class C(Generic[T]): pass
         self.assertEqual(get_args(list | str), (list, str))
         self.assertEqual(get_args(Required[int]), (int,))
         self.assertEqual(get_args(NotRequired[int]), (int,))
+        self.assertEqual(get_args(TypeAlias), ())
```

Comment:
Looks like I added these by resolving the merge. They're not part of PEP 646 but it's good to have these tests in 3.11 too.